cylc/rose/jinja2_parser.py

```diff
@@ -161,10 +161,11 @@ def patch_jinja2_leading_zeros():
     if jinja2.lexer.Lexer.wrap._instances:
         num_examples = 5
         LOG.warning(
-            'Support for integers with leading zeros was dropped'
-            ' in Jinja2 v3.'
+            'Support for integers with leading zeros (including 
+            ' lists of integers) was dropped in Jinja2 v3.'
```

Comment:
```suggestion
            'Support for integers with leading zeros (including 
            ' lists of integers) was dropped in Jinja2 v3.'
```
(Noting my comment on the Issue) maybe slightly different wording?
```suggestion
            'Support for integers with leading zeros was dropped in Jinja2 v3.'
            'This also affects integers and lists of integers in Rose'
            '"template variable" config sections, which define Jinja2 variables.'
```